nova_vm/src/types.rs

```diff
@@ -0,0 +1,17 @@
+mod language;
+mod spec;
+
+pub use language::Value;
+pub use spec::{Base, Reference, ReferencedName};
+
+impl From<Object> for Value {
+    fn from(value: Object) -> Self {
+        todo!()
+    }
+}
+
+#[derive(Debug)]
+pub struct Object;
```

Comment:
**question:** I presume these are intended to become (checked) wrappers around `Value`?